services/core-api/app/api/mines/project_summary/models/project_summary.py

```diff
@@ -0,0 +1,150 @@
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.associationproxy import association_proxy
+
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from app.api.utils.models_mixins import SoftDeleteMixin, AuditMixin, Base
+from app.api.mines.project_summary.models.project_summary_document_xref import ProjectSummaryDocumentXref
+from app.api.mines.documents.models.mine_document import MineDocument
+from app.api.parties.party.models.party import Party
+
+
+class ProjectSummary(SoftDeleteMixin, AuditMixin, Base):
+    __tablename__ = 'project_summary'
+
+    project_summary_guid = db.Column(
+        UUID(as_uuid=True), primary_key=True, server_default=FetchedValue())
+    project_summary_id = db.Column(
+        db.Integer, server_default=FetchedValue(), nullable=False, unique=True)
+    project_summary_description = db.Column(db.String, nullable=True)
```

Comment:
Since this has a limit in SQL I would suggest adding the limit here or in a validator on the model.